src/Resources/views/CRUD/base_edit_form.html.twig

```diff
@@ -45,8 +47,11 @@
                             </ul>
                             <div class="tab-content">
                                 {% for code, form_tab in admin.formtabs %}
-                                    {% set _tab_name = 'tab_'~admin.uniqid~'_'~loop.index %}
-                                    <div class="tab-pane fade{% if (not app.request.query.has('_tab') and loop.first) or (app.request.query.get('_tab') == _tab_name) %} in active{% endif %}" id="{{ _tab_name }}">
+                                    {% set _tab_name = tab_prefix ~ loop.index %}
```

Comment:
@greg0ire  isn't the _tab_name broken there? it does miss rhe '_' from above, that causes the Tab-JS not work anymore